### DISCUSSION TITLE
Feature suggestion - Command Prompt color switch

### DISCUSSION THREAD

**Discussion by emberaholdon** [AutoDarkMode/Windows-Auto-Night-Mode#533] [Custom scripts]
There is an easy way to change the Command Prompt colors via the registry. I'm using the Command Prompt heavily for my work and it's usually set to white background with black text because it suits the light theme of Windows and it's easier on my eyes. But when the night comes it's more consistent if I switch it to inverse (default) colors.

The registry entry for this is:
HKEY_CURRENT_USER\SOFTWARE\Microsoft\Command Processor

Light theme: "DefaultColor" (DWORD) = f0
Dark theme: "DefaultColor" (DWORD) = 00 (or 07)

Is there a chance you could add Command Prompt to the managed apps?

**Comment by Armin2208** [comment]
nice idea. A white cmd looks nice and I always wanted that!
But much nicer would be a Windows Terminal theme switch. Because Windows Terminal already adapts to the Windows theme, but only the title bar.

**Comment by Spiritreader** [comment]
Due to its simpicity of just setting one registry key and not requiring more complex logic, I will point towards custom scripts for this feature.
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/How-to-add-custom-scripts
